src/AspNetCore/AspNetCore.vcxproj

```diff
@@ -26,7 +26,7 @@
     <ProjectName>AspNetCore</ProjectName>
     <TargetName>aspnetcore</TargetName>
     <LinkIncremental>false</LinkIncremental>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```

Comment:
@jkotalik  Considering ANCM should support Win7 as well, I am not sure that `8.1` is right. Can you confirm if this change does not cause any problem?

Reply:
I'll revert this.